app/views/links/new.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1>Submit Link</h1>
+
+<% if @link.errors.any? %>
+  <ul>
+    <% @link.errors.uniq.each do |error_field| %>
+      <li><%= "#{ error_field.to_s.humanize } " + @link.errors[error_field].join('; ') %></li>
+    <% end %>
+  </ul>
+<% end %>
```

Comment:
Since you have very similar code, aside from the instance variable you're grabbing errors from, between the user forms and link forms, you could pull this code into a partial that expects a `record` or something:
```
<%= render partial: 'shared/model_errors', record: @link %>
```